lua/telescope/builtin/files.lua

```diff
@@ -186,19 +186,24 @@ files.current_buffer_fuzzy_find = function(opts)
 end
 
 files.tags = function(opts)
-  local ctags_file = opts.ctags_file or 'tags'
-
-  if not vim.loop.fs_open(vim.fn.expand(ctags_file), "r", 438) then
-    print('Tags file does not exists. Create one with ctags -R')
-    return
+  local tagfiles
+  if opts.ctags_file then
+    tagfiles = {opts.ctags_file}
+  else
+    tagfiles = vim.fn.map(vim.fn.tagfiles(), 'fnamemodify(v:val, ":p")')
```

Comment:
```lua
local tagfiles = opts.ctags_file 
                        and { opts.ctags_file } 
                        or vim.map....
```

Reply:
I can do the whole `if` block and return from the function from within the right side of the `or` there?

Reply:
No you do that or after. `condition and a or b` is basically just the ternary operator for lua.

Reply:
OK, yeah I saw that that's how lua spells ternary -- that has slightly different behavior then than what's here but guess the minor extra check is fine then.